knpu/ep1-intro/listing-services.md

```diff
@@ -1,16 +1,18 @@
 # Listing and Using Services
 
 Rendering a template is pretty common, so there's a shortcut when you're in a controller.
-Replace all of this code with a simple `return $this->render`.
+Replace all of this code with a simple `return $this->render`:
+
+[[[ code('6ad09ec8e1') ]]]
 
 That's it. Make sure this works by refreshing.
 
 So what does this magic-looking `render()` function actually do? Let's find out!
-Hold command or control (depending on your OS) and click `render` to be taken straight
+Hold command or control (depending on your OS) and click `render()` to be taken straight
 into the base `Controller` class where this function lives: deep in the heart of Symfony.
 
```

Comment:
@weaverryan I think we need show `Controller::render()`'s content here, but I don't know what's the better way to do it. There's no any `composer.json` file in this repo. Should we create one in the `tutorial` folder and add Symfony 3 dependencies to it?